app/Http/Controllers/NoteController.php

```diff
@@ -23,6 +23,11 @@ public function index()
      */
     public function create()
     {
+        $user = auth()->user();
+        $notesCount = Note::query()->where('user_id', $user->id)->count();
```

Comment:
Is there any chance `$user` could be `null` here? You didn't check. If there is some guarantee in the documentation, maybe mention it here in a comment, otherwise you need to check the return value. (Same with `$notesCount`.)